pom.xml

```diff
@@ -208,6 +185,7 @@
         <configuration>
           <pluginFirstClassLoader>true</pluginFirstClassLoader>
           <compatibleSinceVersion>1.5</compatibleSinceVersion>
+          <minimumJavaVersion>${java.level}</minimumJavaVersion>
```

Comment:
```suggestion
```

Nobody does this. https://github.com/jenkinsci/plugin-pom/pull/133 may do this automatically some day.

Reply:
This is added for compliancy with XSD as suggested by my intelliJ as a required param in `<configuration>`.

Reply:
Well tell your IDE to shut up. :grin:

Reply:
https://github.com/jenkinsci/plugin-pom/blob/0c6a61e618eaa913bd0c8489d12a58c78f0bdfed/pom.xml#L644 already defines this, so IDEA is presumably just not taking inheritance into account.